.gitignore

```
@@ -1,3 +1,4 @@
 **/_build/**
+**/locale/**/*.mo
```

We could simplify the expression with:

```suggestion
*.mo
```

BTW: I expect *.mo is excluded by the Git global ignore file, in your home directory: /cc @SebastianMB-IT 

```text
[davidep@davidep1 ~]$ grep -F .mo ~/.gitignore_global 
*.mo
```